about.html

```diff
@@ -0,0 +1,281 @@
+                    <li><a href="#"><img src="./assets/images/facebook.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./assets/images/twitter.svg" alt=""></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            
+        </div>
+        <div class="container nav__container">
+            <div class="nav-top">
+                <div class="btn-holder">
+                    <button id="open-menu-btn" class="open-menu-btn">
+                        <i class='bx bx-menu'></i>
+                    </button>
+                    <button id="close-menu-btn" class="close-menu-btn">
+                        <i class='bx bx-x'></i>
+                    </button>
+                </div>
+                <a class="nav-title" href="index.html">
+                    <span>
+                        <img src="./assets/images/trans-logo1.png" alt="mylogo">
+                     
+                    </span>
+                </a>
+            </div>
+            <div class="nav-bottom">
+                <ul class="nav-menu">
+                    <li> <a  class="nav-item0" href="about.html">About</a></li>
+                    <li> <a class="nav-item1" href="#">Program</a></li>
+                    <li> <a class="nav-item2" href="#">Join</a></li>
+                    <li> <a class="nav-item3" href="#">News</a></li>
+                    <li> <a class="nav-item4" href="#">Sponsor</a></li>
+                    <li> <a class="Campaign" href="#">CC Campaign</a></li>
+                </ul>
+            </div>
+           
+           
+           
+        </div>
+    </nav>
+
+    <section class="about-header">
+        <article class="container about__container">
+            <article class="about-header-text">
+                <h1 class="about_Ptext">HTML, CSS, & JAVASCRIPT FOR DEVELOPERS </h1>
+                <h2 class="about_Stext">Fundamentals of Web Development</h2>
+
+            </article>
+            <article class="about-description">
+                <p>
+                    Learn the fundamentals of web developmemt with HTML, CSS, and Javascript,
+                    as you create responsive web pages and robus web applications.
+                    Transform you skils and build stunning, user-friednly Studies
+                    that are compatible with desktop computers
+                    , tablets, and mobile phones.
+                    Learn the fundamentals of web developmemt with HTML, CSS, and Javascript,
+                    as you create responsive web pages and robus web applications.
+                    Transform you skils and build stunning, user-friednly Studies
+                    that are compatible with desktop computers
+                    , tablets, and mobile phones.
+                </p>
+            </article>
```

Comment:
- Good job done so far, but please style this section of the page according to the design template. Please stick to the layout, font color, spacing, background color etc. please fix this 🙏